lib/target_web/mailers/email_confirmation_mailer.ex

```diff
@@ -0,0 +1,22 @@
+defmodule TargetWeb.EmailConfirmationMailer do
+  @moduledoc """
+  Email confirmation mailer module. Used to deliver emails to
+  recently registered users so they can confirm their emails.
+  """
+
+  alias PowEmailConfirmation.Phoenix.Mailer
```

Comment:
```suggestion
  alias PowEmailConfirmation.Phoenix.Mailer, as: PowEmailConfirmationMailer
```

Yeah I know... but I'm a fan of big descriptive names